### PR TITLE
fix: Remove null value from deployment of reloader securityContext

### PR DESCRIPTION
**Pull request by mimotej**
SSIA -- Kubeval flagged this as not valid. Not sure if it will break anything.

### CODE REVIEW

File: reloader/base/deployment.yaml [hunk 1]
```diff
@@ -38,5 +38,4 @@ spec:
           failureThreshold: 5
           periodSeconds: 10
           successThreshold: 1
-      securityContext: NULL
```
[the remark this answers is not in the document]
Comment (larsks) [reply reloader/base/deployment.yaml:41]: That is valid YAML, yes, but I don't think it's valid to set `securityContext` to ` null`. If I submit a pod with `securityContext: null`, and then `oc get pod mypod -o yaml`, what I see is:

```
  securityContext:
    seLinuxOptions:
      level: s0:c30,c20
```

...which suggests that setting `securityContext: null` is the same as not setting any `securityContext`.

We should just remove the line.

[the remark this answers is not in the document]
Comment (dystewart) [reply reloader/base/deployment.yaml:41]: Just tested in Quicklab and it looks safe to remove the line

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**